lms/djangoapps/verify_student/management/commands/tests/test_send_verification_expiry_email.py

```diff
@@ -54,13 +54,13 @@ def test_expiry_date_range(self):
         user = UserFactory.create()
         verification_in_range = self.create_and_submit(user)
         verification_in_range.status = 'approved'
-        verification_in_range.expiry_date = datetime.now(UTC) - timedelta(days=1)
+        verification_in_range.expiry_date = datetime.now() - timedelta(days=1)
```

Comment:
This just gets a time zone unaware timestamp; it might fix the test failures, but I suspect it's also triggering warnings about use of time zone naive datetime objects.  I was thinking of a fix more along the lines of `from django.utils.timezone import now` among the imports, then just calling `now()` here.

Reply:
Oh okay, sorry my bad. I will do that